src/mcp/index.ts

```diff
@@ -27,6 +75,52 @@ async function main() {
 }
 
 main().catch((error) => {
-    console.error('Failed to start DevCycle MCP server:', error)
+    console.error('❌ Failed to start DevCycle MCP server')
+    console.error('')
+
+    if (error instanceof Error) {
+        // Check for common error patterns and provide helpful guidance
+        if (
+            error.message.includes('authentication') ||
+            error.message.includes('DEVCYCLE_CLIENT_ID')
+        ) {
+            console.error('🔐 Authentication Error:')
```

Comment:
I don't like the emojis - seems a bit unprofessional IMO.

Reply:
This is a CLI output, I think they are fine for that.